context/app/api/vitessce.py

```diff
@@ -124,7 +124,7 @@
     IMAGE_PYRAMID: {"offsets": OFFSETS_PATH, "image": IMAGE_PYRAMID_PATH, },
 }
 
-SEQFISH_HYB_CYCLE_REGEX = r"HybCycle_\d+"
+SEQFISH_HYB_CYCLE_REGEX = r"((HybCycle_\d+)|(final_mRNA_background))"
```

Comment:
```suggestion
SEQFISH_HYB_CYCLE_REGEX = r"HybCycle_\d+|final_mRNA_background"
```
`|` is low precedence: https://docs.python.org/3.8/howto/regex.html#more-metacharacters